static/scss/results.scss

```diff
@@ -7,13 +7,12 @@
 
 .race th {
   font-weight: normal;
-  padding: 0 0.5rem;
   word-break: break-word;
 }
 
-.race td.candidate,
-.race th.candidate {
-  width: 30%;
+.primary .race td.candidate,
+.primary .race th.candidate {
```

Comment:
Does it need both `.primary` and `.race`?

Reply:
I'm thinking ahead to actual elections, when we'll also have "Party" in the table -- if there's a better way to account for this, let me know.

Reply:
Oh, okay, that makes sense.